src/test/incremental/hashes/closure_expressions.rs

```diff
@@ -95,7 +95,7 @@ pub fn add_type_ascription_to_parameter() {
 }
 
 #[cfg(not(cfail1))]
-#[rustc_clean(cfg="cfail2", except="HirBody")]
+#[rustc_clean(cfg="cfail2", except="HirBody, TypeckTables")]
```

Comment:
It is correct.

Reply:
To say a bit more: Before, we were not (fully) capturing the user-given type annotations for closures in the typeck tables. So adding a type annotation did not change the hash of typeck tables. But now we are, so it is good that adding a type annotation changes the hash of typeck tables, since that indicates that things which consume the typeck tables (like MIR borrow check) will have to re-execute if those annotations change. =)